scipy-1.0/scipy-optimize.tex

```diff
@@ -124,9 +126,21 @@ \subsubsection*{Nonlinear Optimization}
   \label{tab:minimize}
 \end{table}
 
-Some problems may require the use of global optimizers to find the minimum of the objective function over the set of all parameter values, especially if the objective function possesses many local minima that other solvers could get stuck in.
-\texttt{Brute} performs a grid search over the entire parameter range, with the best grid point undergoing further polishing by another solver. However, this approach becomes increasingly inefficient as the problem dimensionality increases, or if the objective function has many extrema.
-\texttt{Basinhopping} \cite{Wales1997} uses random perturbation of the parameter vector, with local minimization. The acceptance of the new coordinates is dictated by the Metropolis test typically used in Monte Carlo simulations.
-\texttt{differential\textunderscore evolution} \cite{Wormington1999,Storn1997} is a stochastic solver that improves by gradual evolution of a population of candidate solutions. In each iteration trial candidates are generated by combination of candidates from the existing population; if the trial candidates represent an improvement then the population is updated. \texttt{differential\textunderscore evolution} does not require that the objective function be differentiable.
+\paragraph{Global Minimization}
+As \texttt{minimize} only seeks a local minimum, some problems require the use of a global optimization routine to find the minimum of the objective function over the set of all decision variable values. % , especially if the objective function possesses many local minima that other solvers could get stuck in.
```

Comment:
I think you can remove the comment and show the hole phrase...

Reply:
A global optimizer is appropriate whenever 1) getting a global minimum or lower local minimum is important enough to justify additional computation and 2) the problem is not convex. It would say it is _especially_ needed if those two things apply and 3) there is particular reason to suspect that stationary points other than the global minimum will attract a local minimizer. The _number_ of local minima (beyond one) is only a part of (3). 

So I think the statement "especially if the objective function possesses many local minima that other solvers could get stuck in." is somewhat over-simplified for the fraction of the audience that doesn't already know why a global minimizer might be needed, and unnecessary for those who already know why a global minimizer might be needed.

Reply:
Matt I actually don't think it is an over simplification. I think the objective function getting stuck in bad local minima actually summarises 1), 2) and 3) in a more accurate way than saying these three points separately. That is:
1) is basically the same;
2) for non convex problems you don't actually need to use global solvers, so it is not a very good justification...
3) Seems to me the same thing as well...